doc/contribute/bin_blobs.rst

```diff
@@ -0,0 +1,201 @@
+.. _bin-blobs:
+
+.. warning::
+
+   This page documents Zephyr's support for binary blobs. The tooling and
+   infrastructure for this purpose are still incomplete, and so this page may
+   refer to future additions.
+
+Binary Blobs
+************
+
+In the context of an operating system that supports multiple architectures and
+many different IC families, some functionality may be unavailable without the
+help of executable code distributed in binary form.
+Binary blobs (or blobs for short) are files containing machine code or data in
+proprietary formats, e.g. without corresponding source code released under an
+OSI approved license.
+
+Zephyr supports the distribution and use of binary blobs via its built-in
+mechanisms, with some important caveats, described in the following sections. It
+is important to note that all the information in this section applies only to
+upstream or vanilla Zephyr. There are no limitations whatsoever (except perhaps
```

Comment:
```suggestion
upstream or vanilla Zephyr hosted at https://github.com/zephyrproject-rtos as of August 2022. There are no limitations whatsoever (except perhaps
```

Reply:
Why do you want to add this? We do not specify "Zephyr" in this manner anywhere else in the doc

Reply:
I tend to agree with @marc-hb. URL has the merit of defining upstream without ambiguity or need for further definition.